lib/v2/reuters/index.js

```diff
@@ -0,0 +1,57 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const navigation = ctx.params.navigation || 'world';
+    const rootUrl = 'https://www.reuters.com';
+    let category;
+
+    if (navigation === 'authors') {
+        category = ctx.params.category || 'reuters';
+    }
+    else {
+        category = ctx.params.category || '';
```

Comment:
```suggestion
    const navigation = ctx.params.navigation ?? 'world';
    const rootUrl = 'https://www.reuters.com';
    let category;

    if (navigation === 'authors') {
        category = ctx.params.category ?? 'reuters';
    }
    else {
        category = ctx.params.category ?? '';
```
If you provide a default value for navigation, you route path should be `/:navigation?/:category?` instead of `/:navigation/:category?`

Reply:
I am not fully understand.  I thought that variable with question mark could be empty when accpting user input. and without a question mark means user has to fill an value.